lib/palantir.js

```diff
@@ -339,7 +339,17 @@ function standardizeName(name) {
  */
 PalantirConnector.prototype.getPrimaryKey = function(modelName) {
   const modelInfo = this._models[modelName];
-  const pkPropertyKey = _.findKey(modelInfo.properties, (prop) => prop.palantir.primaryKey);
+
+  // Filtering fields that do not have Palantir properties
+  const props = modelInfo.properties;
+  const filteredProp = _.reduce(_.keys(props), (acc, key) => {
```

Comment:
use native .reduce and Object.keys

Reply:
changed to use native reduce and Object.keys